PopupBridge/src/main/java/com/braintreepayments/popupbridge/PopupBridge.java

```diff
@@ -139,6 +144,7 @@ public void onBrowserSwitchResult(int requestCode, BrowserSwitchResult result, U
             try {
                 json.put("path", returnUri.getPath());
                 json.put("queryItems", queryItems);
+                json.put("fragment", fragment);
```

Comment:
iOS is omitting the `hash` aka `fragment` key/value pair if it's not defined. Any thoughts on which way we should go?

Reply:
Omitting it sounds good to me. I'll make the change.